doc/.vitepress/config.ts

```diff
@@ -0,0 +1,82 @@
+import { defineConfig } from 'vitepress'
+import pkg from "../package.json";
+import markdownItFootnote from 'markdown-it-footnote'
+
+
+// https://vitepress.dev/reference/site-config
+export default defineConfig({
+    title: "wire",
+    description: "a tool to deploy nixos systems",
+    themeConfig: {
+        search: {
+            provider: 'local'
+        },
+
+        // https://vitepress.dev/reference/default-theme-config
+        nav: [
+            { text: 'Home', link: '/' },
+            { text: 'Guide', link: '/guide/wire' },
+            { text: 'Reference', link: '/reference/cli' },
+            {
+                text: pkg.version,
+                items: [
+                    {
+                        text: 'Changelog',
+                        link: 'https://github.com/wires-org/wire/blob/main/CHANGELOG.md'
+                    }
+                ]
+            }
+        ],
+
+        sidebar: {
+            '/guide/': [
+                {
+                    text: 'Introduction',
+                    items: [
+                        { text: 'What is Wire?', link: '/guide/wire' },
+                        { text: 'Getting Started', link: '/guide/getting-started' }
+                    ]
+                },
+                {
+                    text: 'Features',
+                    items: [
+                        { text: 'Parallelism', link: '/guide/parallelism' },
+                        { text: 'Keys & Secrets', link: '/guide/keys' },
+                        { text: 'Tagging', link: '/guide/tagging' },
+                        { text: 'hive.default', link: '/guide/hive-default' },
+                        { text: 'Magic Rollback', link: '/guide/magic-rollback' },
+                    ]
+                },
+                {
+                    text: 'Going Deep',
```

Comment:
```suggestion
                    text: 'Use cases',
```

Something like this to tell the reader of various common use cases with wire, going deep doesn't exactly communicate this IMO

Reply:
I just want to scaffold space to add more in the future